docs/resources/aws_iam_managed_policies.md

```diff
@@ -0,0 +1,72 @@
+---
+title: About the aws_iam_managed_policies Resource
+platform: aws
+---
+
+# aws\_iam\_policies
+
+Use the `aws_iam_managed_policies` InSpec audit resource to test properties of a collection of AWS IAM Policies.
+
+## Syntax
+
+`aws_iam_managed_policies` Resource returns a collection of IAM Policies and allows testing of that collection.
+
+    describe aws_iam_managed_policies do
+      its('policy_names') { should include('test-policy-1') }
+    end
+
+
+
+This resource allows filtering by scope.
+To list only AWS managed policies, set `Scope` to `AWS`. To list only the customer managed policies in your AWS account, set `Scope` to `Local`. If scope is not supplied `ALL` policies are returned.
+
+See also the [AWS documentation on IAM Policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html).
+
+## Properties
+
+|Property              | Description|
+| ---                  | --- |
+|arns                  | The ARN identifier of the specified policy. |
```

Comment:
Update the descriptions.

Reply:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.Client.get_policy